lib/dataset-tools/augmentation/replace_parameters.js

```diff
@@ -330,7 +330,10 @@ module.exports = class ParameterReplacer {
     _transformValue(sentenceValue, programValue, arg) {
         if (this._requotable) {
             if (arg && arg.name === 'id' && arg.type.isEntity && arg.type.type.endsWith(':Hotel')) {
-                sentenceValue = sentenceValue.slice(0, sentenceValue.indexOf(',')).trim();
+                if (sentenceValue.startsWith('the '))
+                    sentenceValue = sentenceValue.slice('the '.length).trim();
```

Comment:
You should use the "definite article" stuff from langPack instead of this.

Reply:
The "the" bug was coming from English side caused; where requoting was capturing "casablanca" instead of "the casablanca" for example and thus you ended up with excessive "the" everywhere. The bug is fixed and this is for cleanups on datasets generated with that bug.

Reply:
I don't quite understand.

Reply:
his is not for removing lang specific articles from the beginning of parameters. Same token ("the") is repeated regardless of language. I will just do this processing outside of genie and remove this here. That would create less confusion I guess.

Reply:
Yes please.